File: Lab_2/src/Prostokat.java
public class Prostokat extends Czworokat
{

    Prostokat(double bok1, double bok2)
    {
        super(bok1, bok1, bok2, bok2, 90);
    }

    @Override
    public double count_Pole()
    {
        pole = bok1 * bok3;
        pole = Math.round(pole * 10.0) / 10.0;
        return pole;
    }

    @Override
    public String get_name()
    {
        return "Prostokat";
    }
}

File: Lab_2/С++/Prostokat.cpp
#include "Prostokat.h"
    Prostokat::Prostokat(double bok1, double bok2) : Czworokat (bok1, bok1, bok2, bok2, 90)

    double Prostokat::count_Pole()
    {
        pole = bok1 * bok3;
        pole = round(pole * 10.0) / 10.0;
        return pole;
    }

